app.js

```diff
@@ -23,7 +23,9 @@ app.use(
     winstonInstance: logger,
     meta: false,
     msg: 'HTTP {{req.method}} {{req.url}}',
-    expressFormat: true
+    expressFormat: true,
+    blacklistedMetaFields: ['trace'],
+    headerBlacklist: ['authorization', 'cookie']
```

Comment:
Great work, please consider adding these values as config values

Reply:
The blacklisted headers hold sensitive information, I don't think they should be logged regardless of the environment.

Reply:
then add it as a default config value